### PR TITLE
[RELEASE 5.0.6] zenodo record update

### PR DESCRIPTION
**Pull request by wpotrzebowski**
## Description

Updating zenodo record in release automation script

### CODE REVIEW

**Comment by wpotrzebowski** [comment]
@pbeaucage and @caitwolf you can add your orcid entries if you want.

**Comment by krzywon** [comment]
Note - none of this is in `main`, so a parallel PR might be worthwhile.

**Comment by wpotrzebowski** [comment]
@wpotrzebowski to add Iestyn

File: build_tools/release_automation.py [hunk 1]
```diff
@@ -49,6 +49,7 @@
         {'name': 'Alina, Gervaise','affiliation': 'University of Tennessee Knoxville'},
         {'name': 'Attala, Ziggy', 'affiliation': 'STFC - Rutherford Appleton Laboratory'},
         {'name': 'Bakker, Jurrian','affiliation': 'Technical Unviersity Delft'},
+        {'name': 'Beaucage, Peter','affiliation': 'National Institute of Standards and Technology'},
```
Comment (pbeaucage) [inline build_tools/release_automation.py:52]:
```suggestion
        {'name': 'Beaucage, Peter','affiliation': 'National Institute of Standards and Technology', 'orcid': '0000-0002-2147-0728'},
```

add orcid for Beaucage

File: build_tools/release_automation.py [hunk 2]
```diff
@@ -74,6 +76,7 @@
         {'name': 'Rozyczko, Piotr','affiliation': 'European Spallation Source ERIC', 'orcid' : '0000-0002-2359-1013' },
         {'name': 'Snow, Tim','affiliation': 'Diamond Light Source','orcid': '0000-0001-7146-6885'},
         {'name': 'Washington, Adam','affiliation': 'STFC - Rutherford Appleton Laboratory'},
+        {'name': 'Wilkins, Lucas','affiliation': 'STFC - Rutherford Appleton Laboratory'},
         {'name': 'Wolf, Caitlyn','affiliation': 'National Institute of Standards and Technology'}
```
Comment (pbeaucage) [inline build_tools/release_automation.py:80]:
```suggestion
        {'name': 'Wolf, Caitlyn','affiliation': 'National Institute of Standards and Technology', 'orcid': '0000-0002-2956-7049'}
```
add orcid for Wolf

**Review by pbeaucage — APPROVED** [review]
added my ORCID; could not find ORCID entries for Nicolas Martinez or Lucas.  LGTM once suggested changes are committed.

**Comment by wpotrzebowski** [comment]
Iestyn was already added to the list.

**Comment by pbeaucage** [comment]
Going back to what @krzywon said - should we restore the branch and do a parallel PR from `5.0.6_zenodo_update` to `main` before deleting it?  Or will we PR the entire contents of `release_5.0.6` back to main at some point?  Otherwise these changes won't propagate to `main`.

**Comment by wpotrzebowski** [comment]
Sorry, I was too quick with closing this branch. I initially thought we will merge release_5.0.6 to main after release (as we did with previous release branches) but it is probably safer to make a separate PR for main. Thanks @pbeaucage for reminding me about it.

**Pull request merged**